client/src/model/state.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Class representing the global state of the app.
+ *
+ * Injected into components that need to read or write app data.
+ */
```

Comment:
Same comment here about class comments. What is a state going to hold beyond datalayers (if you know)?

Reply:
Done - left a TODO to elaborate when there are more fields.